pkg/manager/manager_test.go

```diff
@@ -896,6 +897,12 @@ var _ = Describe("manger.Manager", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(m.GetAPIReader()).NotTo(BeNil())
 	})
+	It("should provide a client to access runtime object", func() {
```

Comment:
Nit: just to be consistent with other tests
```suggestion
	It("should provide a function to get a REST client for a runtime object", func() {
```